CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.2.0 (2019-07-16)
+
+- Migrated the project to use Poetry for package management.
+- Added a WSGI middleware `kw.platform.wsgi.user_agent_middleware` for delaying or
+  rejecting requests which are not KW-RFC-22 compliant.
+- Created `kw.platform.aiohttp` module and added a middleware for delaying or
+  rejecting requests which are not KW-RFC-22 compliant.
+- Created `kw.platform.requests` module which contains:
+  + monkey patching mechanism which patches the `requests` library to add KW-RFC-22
+    compliant user agent to all HTTP requests
+  + custom `kw.platform.requests.KiwiSession`
+- Using Sphinx to generate documentation and deploying it on
+  [readthedocs.org](https://kiwi-platform-py.readthedocs.io)
```

Comment:
The [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) standard which we say we use above says we should organize these into Changed/Added/Fixed etc. sections.

I'm fine with either doing that, or removing the notice that we follow that standard :D